extensions/wikia/AdEngine/js/lookup/openXBidder.js

```diff
@@ -83,16 +104,22 @@ define('ext.wikia.adEngine.lookup.openXBidder', [
 	function getSlotParams(slotName) {
 		var dfpParams = {},
 			dfpKey,
-			price;
+			price,
+			size;
 
-		price = priceMap[slotName];
+		if (!priceMap[slotName]) {
+			return {};
```

Comment:
Could you add some logging here? e.g.
`log(['getSlotParams', 'No response for the slot', slotName], 'debug', logGroup);`